examples/docker_example/main.go

```diff
@@ -0,0 +1,34 @@
+package main
```

Comment:
Add license header, copy it from one of the other examples and change the year to 2019